packages/libsql-client/src/ws.ts

```diff
@@ -148,6 +164,14 @@ export class WsClient implements Client {
             const hranaRowsPromise = streamState.stream.query(hranaStmt);
             streamState.stream.closeGracefully();
 
+            const isSchemaDatabase = await this.getIsSchemaDatabase();
+            if (isSchemaDatabase) {
```

Comment:
we should await for hranaRowsPromise here before calling waitForLastMigrationJobToFinish

Reply:
Done!